org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -34,8 +35,16 @@
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		Object firstElement = selection.getFirstElement();
+		if (firstElement instanceof IResource) {
```

Comment:
it's possible that some selected elements (such as a GitHub repo) aren't IResource but can be used to populate the fields. In such case, you should also look for `firstElement instanceof IAdaptable && ((IAdaptable)firstElement).getAdapter(IResource.class)` which may return a resource for non-IResource types.